mubench.reviewsite/templates/index.phtml

```diff
@@ -30,7 +31,11 @@ use MuBench\ReviewSite\Models\Experiment;
                 <ul>
                 <?php if ($detectors[$experiment->id]): ?>
                     <?php foreach ($detectors[$experiment->id] as $detector): ?>
-                        <li><a href="<?= $pathFor('experiment.detector', ['experiment_id' => $experiment->id, 'detector_muid' => $detector->muid]) ?>"><?= $detector->muid ?></a></li>
```

Comment:
Please right-align the badges here... I want this to look like a table with a second column showing the icons. Otherwise I image it to be very noisy, if there are many detectors with different names lengths.